### PR TITLE
Add slice examples for reading data from Omero Tables

### PR DESCRIPTION
**Pull request by kkoz**
# What this PR does

Give this PR a descriptive title then expand on that here.
Just adding examples of using the `slice` function to retrieve data from Omero Tables.
The `slice` function is being introduced in `omero-web` in https://github.com/ome/omero-web/pull/243.

# Testing this PR

1. required setup
Setup training examples

2. actions to perform
Run training examples

3. expected observations
Output should be the same as the corresponding existing examples.

# Related reading

Link to cards, tickets, other PRs:
https://github.com/ome/omero-web/pull/243

1. background for understanding this PR

2. what this PR assists, fixes, or otherwise affects

### CODE REVIEW

**Comment by sbesson** [comment]
@kkoz sorry for the noise as we are fixing the CI. Could you merge `origin/develop` into your branch and/or rebase?

**Comment by sbesson** [comment]
@kkoz sorry for the noise as we are updating the per-PR CI instrastructure. I just merged `origin/develop` into your branch.

**Comment by joshmoore** [comment]
Listed for review tomorrow (since only docs), but I notice @sbesson that this is still excluded from the build. Does that mean a different job would need updating?

**Comment by sbesson** [comment]
Thanks for checking @joshmoore. Fixed a configuration bug on merge-ci so this should properly be included tomorrow and opened https://github.com/ome/devspace/pull/176 accordingly

**Comment by joshmoore** [comment]
```
Merged PRs:
  - PR 6176 sbesson 'Restore CLI DB integration tests'
  - PR 6256 jburel 'point to guides'
  - PR 6260 kkoz 'Add slice examples for reading data from Omero Tables'
```
:+1:

File: examples/Training/python/Tables.py [hunk 1]
```diff
@@ -102,6 +102,17 @@
         print("   ", v)
 
 
+# Get data from every column of the specified rows using slice
+# ============================================================
+row_numbers = [3, 5, 7]
+print("\nGet All Data for rows with slice: ", row_numbers)
+data = open_table.slice(range(len(open_table.getHeaders()), row_numbers))
```
Comment (manics) [inline examples/Training/python/Tables.py]: On the IDR
```py
In [24]: oid = 5223331

In [25]: open_table = s.sharedResources().openTable(omero.model.OriginalFileI(oi
    ...: d))

In [26]: row_numbers = [3, 5, 7]

In [27]: data = open_table.slice(range(len(open_table.getHeaders()), row_numbers
    ...: ))
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-27-9685ca9bd501> in <module>
----> 1 data = open_table.slice(range(len(open_table.getHeaders()), row_numbers))

TypeError: 'list' object cannot be interpreted as an integer
```

Comment (pwalczysko) [reply examples/Training/python/Tables.py]: Yes, I can repeat this with a different table on a different server (outreach server).

```
In [10]: oid=98319

In [11]:

In [11]: open_table = session.sharedResources().openTable(omero.model.OriginalFi
    ...: leI(oid)
    ...: )

In [12]: row_numbers = [3, 5, 7]

In [13]: print("\nGet All Data for rows with slice: ", row_numbers)

Get All Data for rows with slice:  [3, 5, 7]

In [14]: data = open_table.slice(range(len(open_table.getHeaders()), row_numbers
    ...: ))
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-14-9685ca9bd501> in <module>
----> 1 data = open_table.slice(range(len(open_table.getHeaders()), row_numbers))

TypeError: 'list' object cannot be interpreted as an integer

In [15]:
```

Comment (pwalczysko) [reply examples/Training/python/Tables.py]: Also confirming that the corresponding non-slice example works

```
In [18]: data = open_table.readCoordinates(list(range(6)))
    ...: for col in data.columns:
    ...:     print("Data for Column: ", col.name)
    ...:     for v in col.values:
    ...:         print("   ", v)
    ...:
    ...:
Data for Column:  ImageNumber
    1.0
    1.0
    1.0
    1.0
...
```

**Comment by sbesson** [comment]
Note https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-training/589/console failed earlier probably for the same underlying reasons as https://github.com/ome/openmicroscopy/pull/6260#discussion_r538205811

**Comment by jburel** [comment]
The upstream PR has been merged and included in 5.9.0. 
Training examples are green see https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-training/945/
Merging

**Pull request merged**